src/explore-education-statistics-common/src/components/Details.tsx

```diff
@@ -100,6 +100,7 @@ const Details = ({
         aria-expanded={onMounted(open)}
         className="govuk-details__summary"
         role={onMounted('button')}
+        // role="button"
```

Comment:
Is this left in deliberately? 

(Also just for my own curiousity - why don't we want this to have the button role until it's fully rendered?)

Reply:
No, it was left over from when I was wrangling with v13 of react testing library which had loads of problems clicking buttons!  I ended up upgrading to v14 instead.

I think it only gets the role on mount so that it works as a standard html details element when javascript is disabled. But I could be wrong, this component has been here longer than me.